independent-publisher-connectors/Clockify/README.md

```diff
@@ -0,0 +1,50 @@
+# Clockify
+
+Clockify is a popular, easy to use, free time tracking software solution. This connector helps to connect your Clockify account to the Power Platform so you can pull data like Workspaces, Projects, Tasks and TimeEntries.
+
+## Publisher: Dr Adrian Colquhoun (Strategik)
+
+## Prerequisites
+
+To use this connector, you need the following
+
+- A Microsoft Power Apps or Power Automate plan with custom connector feature
+- A Clockify account. Signup here : https://clockify.me/
+
+## Obtaining Credentials
+- Configure API Key here : https://clockify.me/user/settings
+
+
+### Supported Actions
```

Comment:
Could you please fix it to be the same as other headers? Also, the file needs to follow our template file, this section should be called Supported Operations. You can follow the official template on this [link](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md).